src/wp-admin/includes/class-wp-filesystem-direct.php

```diff
@@ -484,10 +484,10 @@ public function mtime( $file ) {
 	 * @since 2.5.0
 	 *
 	 * @param string $file Path to file.
-	 * @return int|false Size of the file in bytes on success, false on failure.
+	 * @return int Size of the file in bytes, 0 otherwise.
 	 */
 	public function size( $file ) {
-		return @filesize( $file );
+		return is_readable( $file ) ? (int) filesize( $file ) : 0;
```

Comment:
```suggestion
		return wp_filesize( $file );
```

This seems cleaner.

Reply:
Thanks, @spacedmonkey for the feedback. I have updated the code as per your suggestion.